src/lxc/cmd/lxc-checkconfig.in

```diff
@@ -66,7 +66,8 @@ is_probed() {
     fi
 }
 
-echo "LXC version $(lxc-start --version)"
+# fallback to using `lxc` binary to obtain version (for lxd compatability)
+echo "LXC version $($(command -v lxc-start || command -v lxc) --version)"
```

Comment:
Hmm, no, this is incorrect as it would report the LXD version rather than the LXC version.
As this is a templated file (`.in`), it should just include the build-time version by using `@PACKAGE_VERSION@`

Reply:
Ah, duh, sorry. But since it this script mostly used for debugging, wouldn't it be best to try to get the current running version of `lxc` in case that is for some reason different than what it was when the `lxc-checkconfig` script was compiled, or no?

I think `lxc info|awk '$1 ~ /driver_version/ {print $2}'` would produce the true version, no?

Reply:
I don't think we should have code in the LXC repo which calls binaries from the LXD repo.

Reply:
I'd say calling `lxc-start --version` if it exists and falling back to `@PACKAGE_VERSION@` should be sufficient here to accommodate most setups.